doc/reference/reference_lua/box_stat/net.rst

```diff
@@ -0,0 +1,45 @@
+box.stat.net()
+==============================
```

Comment:
```suggestion
box.stat.net()
==============
```

The underline must be the same length as the header. Check our [markup guide](https://www.tarantool.io/en/doc/latest/contributing/docs/markup/headings/).